.github/workflows/build-base.yml

```diff
@@ -68,15 +49,42 @@ jobs:
           registry: us-central1-docker.pkg.dev
           username: _json_key_base64
           password: ${{ secrets.GAR_JSON_B64 }}
+      - name: Determine base image eups tag
+        run: |
+          if [[ -n "$STACK_TAG" ]]; then
+            if [[ "$STACK_TAG" == "*_latest" ]]; then
+              echo "$STACK_TAG" > lsst.docker.tag
+            else
+              echo "7-stack-lsst_distrib-$STACK_TAG" > lsst.docker.tag
+            fi
+            echo "$STACK_TAG" > stack.tag
+          else
+            echo "d_latest" > lsst.docker.tag
+            echo "d_latest" > stack.tag
+          fi
+          docker run lsstsqre/centos:"$(< lsst.docker.tag)" bash -c "cat stack/miniconda*/ups_db/global.tags" > eups.tag
```

Comment:
The Science Pipelines base containers don't have an appropriate label, as far as I can tell, and I believe retrieving that label would require downloading the whole container anyway.  Since we need the container for the build, downloading it and running a command in it is not much additional expense.

For these containers, because of the way they're built, it is guaranteed that there will only be one tag.